content/en/users/aai/check-in/vos/_index.md

```diff
@@ -305,32 +305,38 @@ Features:
 #### Connection Parameters
 
 <!-- prettier-ignore -->
-{{< tabpane >}}
-{{< tab header="Production environment" lang="shell" >}}
+{{< tabpanex >}}
+{{< tabx header="Production environment">}}
+```shell
```

Comment:
You need to add a blank space before/after the fenced block, see https://github.com/EGI-Federation/documentation/runs/5337813359?check_suite_focus=true#step:4:10786.

Once fixed it should make the test finally pass, thanks for the work and the patience :)

Reply:
Done